static/javascripts/kp_extraction_emb.js

```diff
@@ -1,37 +1,103 @@
 function submit(input_text, number_keyphrases) {
-	$("#last_group").append('<i class="fa fa-spinner fa-spin" id="spinner"></i>');
-	console.log("url input: ", input_text, 'number keyphrases', number_keyphrases);
-	$('#input_text').val('');
-	url = "/kp/emb";
-	data = {'inp_url' : input_text,
-	  'nbkp' : number_keyphrases
-	  };
-	$.ajax({
-	  type: "POST",
-	  url: url,
-	  contentType: 'application/json',
-	  data: JSON.stringify(data, null, '\t'),
-	  success: function(data){
-	  	console.log(data);
-	  	$('#kpvizu_div').html(data);
-	  	$("#spinner").remove();
-	  }
-	});
+    $("#last_group").append('<i class="fa fa-spinner fa-spin" id="spinner"></i>');
+    $('#input_text').val(input_text);
+
+    url = "http://52.30.254.33:1277/api/kp";
+    data = {
```

Comment:
Same for all those variables

Reply:
for which exactly?

Reply:
url, data... Basically all your variables should be defined as const